### PR TITLE
Tatau 133

### PR DESCRIPTION
**Pull request by IgorKalaidjan**
Fix Evaluation

### CODE REVIEW

File: tatau_core/db/bigchaindb.py [hunk 1]
```diff
@@ -6,9 +6,15 @@
 from bigchaindb_driver import BigchainDB
 from bigchaindb_driver.driver import TransactionsEndpoint
 
+from tatau_core.db import exceptions
+
 logger = getLogger('tatau_core')
 
 
+class ReloadRequired(object):
```
Comment (makar21) [inline tatau_core/db/bigchaindb.py]: wtf? :)

File: tatau_core/node/worker/worker_node.py [hunk 2]
```diff
@@ -150,7 +150,12 @@ def _train(self, task_assignment: TaskAssignment):
 
         if task_assignment.task_declaration.is_horovod:
             tflops = self._do_hvd_train(task_assignment)
-            task_assignment = task_assignment.reload()
+            failed, t = self._run_eval_session(task_assignment)
```
Comment (makar21) [inline tatau_core/node/worker/worker_node.py]: rename t to eval_tflops

File: tatau_core/sessions/hvd_train.py [hunk 3]
```diff
@@ -247,6 +247,8 @@ def train(self):
             train_progress=progress, current_iteration=current_iteration
         )
 
+        # TODO Create Evaluation
```
Comment (makar21) [inline tatau_core/sessions/hvd_train.py]: remove this

File: tatau_core/db/bigchaindb.py (hunk 1 above)
Comment (IgorKalaidjan) [reply tatau_core/db/bigchaindb.py]: Done

File: tatau_core/node/worker/worker_node.py (hunk 2 above)
Comment (IgorKalaidjan) [reply tatau_core/node/worker/worker_node.py]: Done

File: tatau_core/sessions/hvd_train.py (hunk 3 above)
Comment (IgorKalaidjan) [reply tatau_core/sessions/hvd_train.py]: Done

**Pull request merged**